create-gcp-resources.sh

```diff
@@ -40,6 +44,11 @@ gcloud compute instances create "${INSTANCE}" \
   --network "${INSTANCE}" \
   --subnet "${INSTANCE}"
 
+# pause for ssh for a few seconds
+echo pause for 10s before copying the pull-secret
+sleep 10s
```

Comment:
@sallyom Are we sure that within 10s the ssh command is available for use? Can we poll with ssh command with a timeout around 30-40sec?

Reply:
yes, I was meaning to update that, you're right, this last cmd fails even w/ the 10s - i'll update it

Reply:
added, done